src/ResourceManagement/OperationalInsights/Microsoft.Azure.Management.OperationalInsights/Generated/Models/ProxyResource.cs

```diff
@@ -0,0 +1,59 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.OperationalInsights.Models
+{
+    using System.Linq;
+
+    /// <summary>
+    /// Common properties of proxy resource.
+    /// </summary>
+    public partial class ProxyResource
```

Comment:
I created this ProxyResource class because "location" field has a different constraint. If you look at the last comments thread in my swagger PR, you will see that I made "location" a required property for tracked resource but not required for ProxyResource per the reviewers' suggestion. Let me if that is wrong.